CMakeLists.txt

```diff
@@ -1014,6 +1015,10 @@ if(DEFINED WITH_INTELPERC)
   status("    Intel PerC:"     HAVE_INTELPERC      THEN "YES"                                 ELSE NO)
 endif(DEFINED WITH_INTELPERC)
 
+if(DEFINED WITH_GPHOTO2)
+  status("    gPhoto2:"          HAVE_GPHOTO2          THEN "YES"                                        ELSE NO)
```

Comment:
Could you try aligning the `HAVE_GPHOTO2` with the above `HAVE_INTELPERC`, this is just a beauty fix but one many programmers like. Same for the other parts of that line.